autonomy/replay/utils.py

```diff
@@ -27,7 +27,7 @@
 def fix_address_books(build_dir: Path) -> None:
     """Update address books in data dump to use them in replays."""
     for addr_file in sorted(
-        (build_dir / "persistent_data" / "dumps").glob("**/addrbook.json")
+        (build_dir / "persistent_data" / "tm_state").glob("**/addrbook.json")
```

Comment:
All these magic values need to be defined as importable constants once and then reused